source/audio/appsVolume.py

```diff
@@ -163,7 +163,15 @@ def _toggleAppsVolumeState():
 	state = _APPS_VOLUME_STATES_ORDER[index]
 	config.conf["audio"]["applicationsVolumeMode"] = state.name
 	_updateAppsVolumeImpl(volume / 100.0, muted, state)
-	ui.message(state.displayString)
+	if state == AppsVolumeAdjusterFlag.ENABLED:
+		# Translators: Reported as a result of the command to allow or disallow to control the volume of other
```

Comment:
```suggestion
		# Translators: Reported as a result of toggling the availability of the other apps volume adjuster.
```

Reply:
The notion of "volume adjuster" was unclear and had been removed from audio settings panel for another wording. The goal of this PR is to remove "volume adjuster" where it was remaining in the GUI, and thus also in translators comments.

Thus I'll not take this suggestion; I'll commit something else.